src/lambdas/common-platform-report/src/SendReportUseCase.ts

```diff
@@ -1,30 +1,28 @@
 /* eslint-disable no-console */
-import type { DynamoGateway } from "@bichard/dynamo-gateway"
 import type { PromiseResult } from "@bichard/types"
 import { isError } from "@bichard/types"
 import config from "./config"
 import generateDates from "./generateDates"
-import getReportData from "./getReportData"
+import getReportDataApi from "./getReportDataApi"
```

Comment:
I think we could just keep the same name and just replace the contents of the existing `getReportData` module